testkit/src/main/scala/org/bitcoins/keymanager/KeyManagerTestUtil.scala

```diff
@@ -15,5 +19,24 @@ object KeyManagerTestUtil {
       .resolve(WalletStorage.ENCRYPTED_SEED_FILE_NAME)
   }
 
+  def createKeyManagerParams(): KeyManagerParams = {
+    val seedPath = KeyManagerTestUtil.tmpSeedPath
+    KeyManagerParams(seedPath = seedPath,
+                     purpose = Gen.oneOf(HDPurposes.all).sample.get,
+                     network = Gen.oneOf(Networks.knownNetworks).sample.get)
+  }
+
+  def bip39PasswordOpt: Option[String] = {
+    if (scala.util.Random.nextBoolean()) {
+      Some(bip39Password)
+    } else {
+      None
+    }
+  }
+
+  def bip39Password: String = {
+    CryptoGenerators.bip39Password.sample.get
```

Comment:
you should use the `GenUtil` sample here in case `suchThat` is ever introduced or anything like that